.github/workflows/pipeline.yml

```diff
@@ -38,7 +38,7 @@ jobs:
             ./packages/rust-bindings/pkg
             ./packages/common/lib
             ./packages/nodejs/lib
-          key: ${{ runner.os }}-build_output-${{ hashFiles('**/*.ts', '**/yarn.lock', '.git/modules/deps/concordium-base/HEAD', './packages/rust-bindings/Cargo.lock', '.packages/rust-bindings/src/*.rs') }}
+          key: ${{ runner.os }}-build_output-${{ hashFiles('.git/refs') }}
```

Comment:
Are you sure this is the same `/refs` contains a whole bunch of crap. Typically here we want to explicitly list the files that have changed, to be explicit about it.

Using `.git/refs` seems to, if it works, to work by accident.

Reply:
For the build-output specifically, we would want to only cache it for jobs that depend on it, because any meaningful change you make to the SDK would require a rebuild anyways, so the question to ask is: "Are we on a unique commit?". 

This means I essentially just wanted the HEAD of the current commit ref. I have later found out that we can do `env.GIT_SHA` which gives me what I originally wanted.